platform/darwin/src/MGLLocationManager.h

```diff
@@ -0,0 +1,103 @@
+#import <Foundation/Foundation.h>
+#import <CoreLocation/CoreLocation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol MGLLocationManagerDelegate;
+
+/**
+ The `MGLLocationManager` protocol defines a set of methods that you
+ use to receive location-related events.
+ */
```

Comment:
We can mention that CLLocationManager conforms to MGLLocationManager, but that the developer may also implement a location manager that isn’t a kind of CLLocationManager.